demisto_sdk/commands/content_graph/objects/repository.py

```diff
@@ -17,25 +18,41 @@
 USE_MULTIPROCESSING = False  # toggle this for better debugging
 
 
+@lru_cache
+def from_path(path: Path = CONTENT_PATH, packs_to_parse: Optional[Tuple[str]] = None):
```

Comment:
why isn't it a class method of `ContentDTO`?

Reply:
I documented, for some reason I get cache miss whin it's in the class.